src/App.tsx

```diff
@@ -1,35 +1,69 @@
-import { useState } from 'react'
+import React, { Component } from 'react';
+import * as THREE from 'three';
 import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import viteLogo from './vite.svg'
 import './App.css'
+import { OBJLoader } from 'three/addons/loaders/OBJLoader.js';
+import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
 
-function App() {
-  const [count, setCount] = useState(0)
-
-  return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+
+class App extends Component {
+  componentDidMount() {
+    const scene = new THREE.Scene();
+    const camera = new THREE.PerspectiveCamera(45, window.innerWidth / window.innerHeight, 0.1, 20);
+    const renderer = new THREE.WebGLRenderer();
+
+    camera.position.set(0, 0, 5);
+    let rotationSpeed = 0.01;
+    
+    renderer.setSize(window.innerWidth, window.innerHeight);
+    document.body.appendChild(renderer.domElement);
+      
+    const ambientLight = new THREE.AmbientLight( 0xffffff );
+    scene.add( ambientLight );
+
+
+    /////
+    const lineMaterial = new THREE.LineBasicMaterial({ color: 0x00ff00 });
+    const lineGeometry = new THREE.BufferGeometry();
+    lineGeometry.setFromPoints([new THREE.Vector3(0, -window.innerHeight / 2, 0), new THREE.Vector3(0, window.innerHeight / 2, 0)]);
+    const line = new THREE.Line(lineGeometry, lineMaterial);
+    scene.add(line);
+    /////
+
+    let loader = new OBJLoader();
+    let object3D: THREE.Object3D;
+
+    loader.load(
+      '/Cerberus.obj',
+      function (object) {
+        object3D = object;
+        scene.add(object);
+      }
+    );
+      
+    function handleMouseMove(event: MouseEvent) {
+      const mouseX = (event.clientX / window.innerWidth) * 2 - 1;
```

Comment:
Please do not use magic numbers. Extract them to constants.

Reply:
I declared variables for every number